web/src/invite.ts

```diff
@@ -308,6 +309,33 @@ function set_streams_to_join_list_visibility(): void {
     }
 }
 
+function update_guest_visible_users_count(): void {
+    const invite_as_value = $("#invite_as").val();
+
+    if (typeof invite_as_value !== "string") {
+        return;
```

Comment:
A better way to handle this would be how the `invite_as` value is fetched in `get_common_invitation_data` function in the same file.